common/app/views/fragments/contentFooter.scala.html

```diff
@@ -19,7 +19,7 @@
     } {
         <div class="js-fronts-network-2" data-link-name="fronts-on-articles | network-front | block-2"></div>
     } {
-        @if(content.commercial.hasHighMerchandisingTarget) {
+        @if(content.commercial.hasHighMerchandisingAdUnitAndTag) {
```

Comment:
It still has an odd name, for someone reading the content footer. A higher-level name would hide the implementation, and reveal the intent. For example `needsHighMerchandisingSlot`